backend/reservations/reporting.py

```diff
@@ -139,3 +139,69 @@ def hotel_export(hotel):
 
     egest_csv(rows, fields, hotel_export_file)
     return hotel_export_file
+
+
+def rooming_list_export(hotel):
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    guests = Guest.objects.filter(name_hotel = hotel.title())
+    if rooms.count() == 0:
+        raise Exception("No rooms found for hotel %s" % hotel)
+    
+    cols = [
+        "room",
+        "room_type",
+        "first_name",
+        "last_name",
+        "secondary_name", 
+        "check_in_date",
+        "check_out_date",
+        "is_comp",
+        "sp_ticket_id",
+    ]
+    
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    if rooms.count() == 0:
+        raise Exception("No rooms found for hotel %s" % hotel)
+
+    rows = []
+    for room in rooms:
+        # hacky split to first/last name
+        primary_name = room.primary.split(" ", 1)
+        first_name = primary_name[0]
+        last_name = primary_name[1] if len(primary_name)>1 else ""
+        row = {
+            'room_number': room.number,
+            'room_type': room.hotel_sku(),
+            'first_name': first_name,
+            'last_name': last_name,
+        }
+        if room.secondary != '':
+            row['secondary_name'] = room.secondary
+        if room.check_in and room.check_out:
```

Comment:
Oh this makes me think. At this point, every room should have an _actual_ date for `check_in` / `check_out`. Will add a check for this to `find_drama` ( #143 )